docs/examples/config/example-config.json

```diff
@@ -67,7 +67,7 @@
         "local_amdin": false
     },
     // Time in seconds for graceful shutdown. Defaults to 10 seconds. Not fully implemented yet.
-    "graceful_period": 10.,
```

Comment:
It would be better to change it to `10.0` to show that it is floating point value